styles/main.scss

```diff
@@ -50,8 +50,9 @@ a:hover {
 }
 
 hr {
-  height: 2px;
-  background: #616161;
+  height: 1px;
+  border: none;
+  background: #e8e8e8;
```

Comment:
@ntorbinskiy This one has a dark version as well
![image](https://user-images.githubusercontent.com/46556479/198589844-ce556bfb-4199-4228-a549-c720648618fc.png)